vtex-apps/b2b-quotes
# Update pt.json

Translation to PT-BR

#### What problem is this solving?
Translation of all labels and messages to BR-PT language.


#### How to test it?
Navigate the app and check that the messages are translated.


[Workspace](Link goes here!)

#### Screenshots or example usage:



#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](put .gif link here - can be found under "advanced" on giphy)
